src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/scanner/FuzzYAMLRead50431Test.kt

```diff
@@ -0,0 +1,41 @@
+package it.krzeminski.snakeyaml.engine.kmp.scanner
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.common.Platform
+import io.kotest.common.platform
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContain
+import it.krzeminski.snakeyaml.engine.kmp.api.Load
+import it.krzeminski.snakeyaml.engine.kmp.exceptions.ScannerException
+
+class FuzzYAMLRead50431Test : FunSpec({
+
+    val load = Load()
+
+    test("incomplete value should throw ScannerException") {
+        shouldThrow<ScannerException> {
+            load.loadOne("\"\\UE30EEE")
+        }.also {
+            // There's a difference in the error message between JVM and other platforms.
+            // TODO: understand and fix in https://github.com/krzema12/snakeyaml-engine-kmp/issues/550
```

Comment:
To be investigated in https://github.com/krzema12/snakeyaml-engine-kmp/issues/550